### PR TITLE
update tfx cab sample links

### PR DESCRIPTION
**Pull request by gaoning777**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1157)

### CODE REVIEW

File: content/docs/pipelines/sdk/output-viewer.md [hunk 1]
```diff
@@ -437,9 +437,8 @@ You can run the sample by selecting
 Kubeflow Pipelines UI. For help getting started with the UI, follow the 
 [Kubeflow Pipelines quickstart](/docs/pipelines/pipelines-quickstart/).
 
-The sample code is available in the [Kubeflow Pipelines samples 
-repo](https://github.com/kubeflow/pipelines/tree/master/samples/core/tfx_cab_classification). The
-pipeline uses a number of prebuilt, reusable components, including:
+# Will replace the tfx cab with tfx oss when it is ready.
```
Comment (sarahmaddox) [inline content/docs/pipelines/sdk/output-viewer.md]: Remove this line. It appears in the rendered page.
https://deploy-preview-1157--competent-brattain-de2d6d.netlify.com/docs/pipelines/sdk/output-viewer/#will-replace-the-tfx-cab-with-tfx-oss-when-it-is-ready

Comment (gaoning777) [reply content/docs/pipelines/sdk/output-viewer.md]: Done.

**Pull request merged**